playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -231,6 +231,24 @@
     - reconcile_jenkins_role_binding_result.rc == 0
     when: (not openshift.common.version_gte_3_7 | bool) and (openshift.common.version_gte_3_4_or_1_4 | bool)
 
+  - copy:
+      src: "{{ __master_files_location }}/{{ item }}"
+      dest: "{{ tmpdir }}/{{ item }}"
+    with_items:
+    - "{{ __master_shared_resource_viewer_file }}"
+    when: (openshift.common.version_gte_3_6 | bool) and (not openshift.common.version_gte_3_7 | bool)
```

Comment:
Could use a `block` structure here and place both of these tasks underneath it.

Reply:
See how it looks now @kwoodson